### PR TITLE
Initial proposition for PHP scaffold

### PR DESCRIPTION
**Pull request by Malian**
@poulinjulien @jbruggem @toch @tab87vn

### CODE REVIEW

**Comment by jbruggem** [comment]
Great to see a new language arrive ! 

I would very much like to see a version where everything can be run inside a docker container so that installing packages is not needed.

File: scaffolds/php/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# PHP Scaffold for Code Retreat
+
+## Install PHP tools
```
Comment (jbruggem) [inline scaffolds/php/README.md]: It's missing instructions to install PHP itself :)

File: scaffolds/php/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+You need to install a package manager for PHP called _composer_.
+Execute the following code to install it in this local repository:
+``` bash
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```
Comment (jbruggem) [inline scaffolds/php/README.md]: Isn't there an ubuntu package this ? Wouldn't it be preferable ?

File: scaffolds/php/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+php -r "if (hash_file('SHA384', 'composer-setup.php') === '544e09ee996cdf60ece3804abc52599c22b1f40f4323403c44d44fdfdd586475ca9813a858088ffbc1f233e9b180f061') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php --install-dir=bin --filename=composer
+php -r "unlink('composer-setup.php');"
+```
+
+Follow thie [link](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) for more details or if you want to install it globaly.
```
Comment (poulinjulien) [inline scaffolds/php/README.md]: 'thie' => 'the'

File: scaffolds/php/README.md (hunk 2 above)
Comment (Malian) [reply scaffolds/php/README.md]: Nop. I never have seen that before.

**Comment by Malian** [comment]
> Great to see a new language arrive !
> 
> I would very much like to see a version where everything can be run inside a docker container so that installing packages is not needed.

Is it better?

File: scaffolds/php/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# PHP Scaffold for Code Retreat
+
+This scaffold comes with severals tools that work with docker under the hood.
+- `php`: a php 7.1 executable
+- `phpspec`: a php test framework
+- `composer`: a php dependencies manager
+
+Run
+```bash
+docker.sh [command]
```
Comment (toch) [inline scaffolds/php/README.md]: You probably don't have the current directory `.` in your `PATH`, and it's good.
As consequence, isn't it `./docker.sh`?

File: scaffolds/php/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+```
+to use a tool. 
+
+## Install or update dependencies
+
+The first time you use this scaffold you need to install all dependencies. This is done by using the php dependencies manager: [composer](https://getcomposer.org/). The following command should install all the dependencies in the `/vendor` directory.
```
Comment (toch) [inline scaffolds/php/README.md]: keep your lines shorter than 100 characters. Better for diff. Thanks.

Comment (toch) [inline scaffolds/php/README.md]: Upcase `PHP`?

File: scaffolds/php/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Sometimes you need to update your dependencies or add a new one. As an example, we will add a second php unit test framework called: _phpunit_.
+```bash
+./docker.sh composer require phpunit/phpunit ^6.2
+```
+
+That's it! You can now use your favorite tools!
+
+## Run the tests
+
+[phpspec](http://www.phpspec.net/en/stable/manual/introduction.html) is a tool which can help you write clean and working PHP code using behaviour driven development. Run the following command to execute tests:
```
Comment (toch) [inline scaffolds/php/README.md]: :heart:

File: scaffolds/php/README.md (hunk 4 above)
Comment (Malian) [reply scaffolds/php/README.md]: Good catch!

**Pull request merged**